### PR TITLE
Updating the list of skus

### PR DESCRIPTION
**Pull request by nlupica1**
This pull request updates the list of skus to include Windows Server 2022 and other Windows server skus

### CODE REVIEW

File: samples/Compute/enforce-hybrid-use-benefit/azurepolicy.rules.json [hunk 1]
```diff
@@ -21,16 +21,77 @@
...
+					"2016-datacenter-gensecond",
+					"2016-datacenter-gs",
 					"2016-Datacenter-Server-Core",
+					"2016-datacenter-server-core-g2",
 					"2016-Datacenter-Server-Core-smalldisk",
+					"2016-datacenter-server-core-smalldisk-g2",
 					"2016-Datacenter-smalldisk",
+					"2016-datacenter-smalldisk-g2",
 					"2016-Datacenter-with-Containers",
-					"2016-Datacenter-with-RDSH"
```
Comment (pilor) [inline samples/Compute/enforce-hybrid-use-benefit/azurepolicy.rules.json:33]: why was the -rdsh sku removed?

**Review by pilor — CHANGES_REQUESTED** [review]

File: samples/Compute/enforce-hybrid-use-benefit/azurepolicy.rules.json (hunk 1 above)
Comment (nlupica1) [reply samples/Compute/enforce-hybrid-use-benefit/azurepolicy.rules.json:33]: I used the following command to list all the Windows Server skus: 

`Get-AzVMImageSku -Location "East US" -PublisherName "MicrosoftWindowsServer" -Offer "WindowsServer" | Select Skus`

That sku is not listed in the export.  Is that still a valid sku?  I can re-add it.

**Review by pilor — APPROVED** [review]

**Comment by pilor** [comment]
Please sign the CLA linked in the microsoft-cla comment above

**Comment by nlupica1** [comment]
> Please sign the CLA linked in the microsoft-cla comment above

All set.

**Pull request merged**